ocp_install_env.sh

```diff
@@ -103,8 +103,13 @@ function baremetal_network_configuration() {
 cat <<EOF
     provisioningNetwork: "${PROVISIONING_NETWORK_PROFILE}"
     provisioningHostIP: "${CLUSTER_PROVISIONING_IP}"
+EOF
+    version=$(openshift_version | sed "s/\.//")
+    if [[ $version < 47 ]]; then
+cat <<EOF
```

Comment:
@zaneb pointed out in my PR that the `[[` comparison is wrong and actually compares strings (although that comparison does work in this case), I reworked it to use a `sort` version check instead, or this could be changed to `((`

Reply:
@asalkeld could you replace the string comparison with either a version comparison like in `dnsvip()` or replace the `[[` with `((` please?

Reply:
done, sorry I forgot about this